scylla-cql/src/types/serialize/writers.rs

```diff
@@ -135,37 +189,111 @@ impl<'buf> CellWriter<'buf> {
 /// data to be misinterpreted.
 pub struct CellValueBuilder<'buf> {
     // Buffer that this value should be serialized to.
-    buf: &'buf mut Vec<u8>,
+    pub(crate) buf: &'buf mut Vec<u8>,
 
     // Starting position of the value in the buffer.
     starting_pos: usize,
+
+    // If writing to a fixed length type vector, the type length.
+    cell_len: Option<usize>,
+
+    //If serializing a variable length vector cell, the size is encoded as a varint.
+    is_variable_length: bool,
+
+    // Buffer for variable length vector cell.
+    variable_length_buffer: Option<Vec<u8>>,
 }
 
 impl<'buf> CellValueBuilder<'buf> {
     #[inline]
-    fn new(buf: &'buf mut Vec<u8>) -> Self {
+    fn new(buf: &'buf mut Vec<u8>, size_as_uvar_int: bool) -> Self {
         // "Length" of a [bytes] frame can either be a non-negative i32,
         // -1 (null) or -1 (not set). Push an invalid value here. It will be
```

Comment:
:camping: This looks like a typo in the comment: `-1` is mentioned twice.
According to [the CQL specs](https://github.com/apache/cassandra/blob/57e5cdfb59485d70f6b340987957fa4301233e8c/doc/native_protocol_v4.spec#L229), _not set_ is represented using `-2`.

Could you please fix this as a bonus, @smoczy123?

Reply:
This typo is present in more than one place in this file.